lib/ioutil/log_entry.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ioutil
+
+import (
+	"time"
+)
+
+const logEntryVersion = 1
+
+// LogEntry attaches a timestamp to a log entry
+// easyjson:json
+type LogEntry struct {
+	Log     string    `json:"log"`
+	Time    time.Time `json:"time"`
+	Version int       `json:"version"`
```

Comment:
@vburenin so far, we don't upgrade container, so for one container we safely could only have one version inside.
At this time, we still need to work with the plain text log file, so it should rely on the plugin version defined in pkg/version/version.go. In the future, if we start to support container upgrade, we could add version per entry at that time.